raster/src/main/scala/geotrellis/raster/histogram/FastMapHistogram.scala

```diff
@@ -58,9 +60,9 @@ object FastMapHistogram {
   * The [[FastMapHistogram]] class.  Quickly compute histograms over
   * integer data.
   */
-class FastMapHistogram(_size: Int, _buckets: Array[Int], _used: Int, _total: Int)
+class FastMapHistogram(var size: Int, var buckets: Array[Int], var counts: Array[Long], var used: Int, var total: Long)
```

Comment:
Also your commits are not registered with the github account: https://github.com/geotrellis/geotrellis/pull/1455/commits/8d517fea7dc2d6869815bd4262ec4706500d55a8